projects/packages/changelogger/src/AddCommand.php

```diff
@@ -242,11 +242,24 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
 					$type = strtolower( $type );
 				}
 				if ( $isInteractive ) {
-					$question = new ChoiceQuestion( 'Type of change.', $types, $type );
-					$type     = $this->getHelper( 'question' )->ask( $input, $output, $question );
-					if ( null === $type ) { // non-interactive.
-						$output->writeln( 'Got EOF when attempting to query user, aborting.', OutputInterface::VERBOSITY_VERBOSE ); // @codeCoverageIgnore
-						return 1;
+					$question     = new ChoiceQuestion(
+						'Is this a change users would want to know about?',
```

Comment:
What if we added "self-hosted" to kind of exclude WoA and simple? I guess that's very Automattic-specific though, so not really appropriate for this package.

```suggestion
						'Is this a change self-hosted users would want to know about?',
```

What about "site admins"? Would it be more appropriate?


```suggestion
						'Is this a change site admins would want to know about?',
```

Reply:
I like site admins, I'll plug that in for now.